python/cuml/utils/input_utils.py

```diff
@@ -40,7 +40,7 @@ def get_cudf_column_ptr(col):
     """
     Returns ctype pointer of a cudf column
     """
-    return cudf.bindings.cudf_cpp.get_column_data_ptr(col._column)
+    return cudf._lib.cudf.get_column_data_ptr(col._column)
```

Comment:
Longer term, should we eliminate calls to internal-looking `_lib`?

Reply:
yes, will use cuda_array_interface in the (pretty soon) upcoming PRs about memory structures :)